assets/app/styles/_core.less

```diff
@@ -648,6 +648,11 @@ label.checkbox {
   margin-left: 3px;
 }
 
+.image-summary-description {
```

Comment:
I'm wondering if we should have a more generic class for this, seems like something we will need to use again